### ISSUE TITLE
Official Implementation

### ISSUE THREAD

**Issue by LavenderLA** [ykkawana/neural_star_domain#1]
Hello and thanks for your great work on NSDN! Is this repository an official and mature implementation of your NIPS paper "Neural Star Domain as Primitive Representation"?

**Comment by ykkawana** [comment]
Yes, this is the official implementation repo of my paper at NeurIPS 2020. Thanks.

**Comment by LavenderLA** [comment]
Thanks for your reply! And I have another four questions:

1. In this repo, which file is used to generate separate primitive meshes? Like the plane result in figure 1 in the paper. I found that the `generate.py` file generate the whole mesh as OccNet.
2. How many epochs have you trained to obtain the satisfying result and how long does the training procedure roughly take?
3. Are settings in `nsd_n10.yaml` the best option? 
4. Would you please share the trained checkpoint file of the model?

I would really appreciate it if you could kindly answer these questions. Thanks again for your wonderful work!
